index.html

```diff
@@ -231,6 +231,24 @@ <h4 class="txt-lg">Skills</h4>
     </ul>
 
   </section>
+
+  <section class="contact">
+
+    <h1 class="txt-ctct-me">Contact me</h1>
+    <p class="ctct-me">If you have an application you are interested in developing, a feature that you need built or a
+      project that
+      needs coding. I’d love to help with it </p>
+    <form class="container-form" action="https://formspree.io/f/mrgjqpvb" method="post">
+
+      <input class="text-area" type="text" name="name" value="Yeremias" maxlength="30" required placeholder="name">
+      <input class="text-area" type="email" name="email" value="natajayanj14@gmail.com" required placeholder="email">
+      <textarea class="text-area large" placeholder="Write your message" maxlength="500" required></textarea>
+      <input class="btn-container contact-btn" type="submit" name="submit" value="Get In Touch">
+
+    </form>
```

Comment:
- [ ] [OPTIONAL] Kindly ensure that the input fields and text area are aligned at the center with equal minimal margin to the right and left as provided in the design template. See the image above for more details.
- [ ] [OPTIONAL] Kindly add more right and left padding to the "Get in touch button" as it currently looks smaller than the one provided in the template.

Reply:
Okay. Thank you for your feedback. I will work on it.

Reply:
You are welcome.